spec/compiler/formatter/formatter_spec.cr

```diff
@@ -1610,4 +1610,8 @@ describe Crystal::Formatter do
       # second comment
     }
     CODE
+
+  # 7272
```

Comment:
There's no need to reference the issue, a textual description is better:
```suggestion
  # format expressions in `# =>` comments
```

Reply:
I did it this way because a comment just like that exists in several other places in the file and I wanted to be consistent. However, I do agree that a textual description is superior and I implemented that along with your other suggested changes.

Reply:
FYI commenting in that style is typical only for specs that get added specifically in response to a newly found bug.